example/src/main/java/io/netty/example/http2/helloworld/multiplex/server/Http2ServerInitializer.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2014 The Netty Project
```

Comment:
Done. Note that this is a copy of an existing file, with effectively only line 48 changed. (I was disappointed that GitHub didn't show it as a copy.) Many of the other example files are similar. I've updated the year for them as well.